doc/.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,26 @@
+<!-- Thanks for reporting an issue to axe-core. Please provide all necessary info to reproduce the issue. Without adequate details, your issue may be closed without investigation. -->
+
+## Issue Description
```

Comment:
I'm not too keen on templates like this. I find them a bit off putting when I see them in other projects. Also, we already have a template for new rules. I think this will confuse things. I'd rather we just give some general instructions. I would suggest we add a basic comment at the top, something like:

> If you're reporting a bug, include a description of the issue and a page or code snippet where it can be reproduced. Please make sure you have tested this with the latest version of axe-core. When proposing a new rule, please use our [rule template](#).

Reply:
I do think we should mention accessibility supported in the issue template though–more often than not that's where we have to bring it up. Also, reminding them to include a demo.